tests/test_job.py

```diff
@@ -134,92 +130,10 @@ def test_get_res_run_async():
-    _SLEEP_SEC = 0.2
-
-    def fetch_platform_details(self):
-        time.sleep(self._SLEEP_SEC)
-        return json.dumps({
-            'created_date': 'Mon, 31 Oct 2022 16:54:45 GMT',
-            'description': 'Mocked Simulator',
-            'id': 'e576e49c-7b1a-470b-5910-c04e406d40f6',
-            'jobs': 6687,
-            'name': 'mocked:platform',
-            'perfs': {},
-            'specs': {
-                'available_commands': ['probs'], 'connected_input_modes': [0, 2, 4, 6, 8],
-                'constraints': {
-                    'max_mode_count': 12,
-                    'max_photon_count': 5,
-                    'min_mode_count': 1,
-                    'min_photon_count': 1
-                },
-                'description': 'Simulator of sim:ascella qpu',
-                'detector': 'threshold',
-                'parameters': {
-                    'HOM': 'indistinguishability value, using HOM model (default 1)',
-                    'backend_name': 'name of the backend that will be used for computation (default "SLOS")',
-                    'final_mode_number': 'number of modes of the output states. states having a photon on unused modes will be ignored. Useful when using computed circuits (default input_state.m)',
-                    'g2': 'g2 value (default 0)',
-                    'mode_post_select': 'number of required detected modes to keep a state. (default input_state.n)',
-                    'phase_imprecision': 'imprecision on the phase shifter phases (default 0)',
-                    'transmittance': 'probability at each pulse that a photon is sent to the system and is detected (default 1)'
-                },
-            },
-            'status': 'available',
-            'svg': '',
-            'type': 'simulator',
-            'waiting_jobs': 0
-        })
-
-    def create_job(self, payload):
-        time.sleep(self._SLEEP_SEC)
-        return self._ARBITRARY_JOB_ID
-
-    def cancel_job(self, job_id: str):
-        time.sleep(self._SLEEP_SEC)
-
-    def get_job_status(self, job_id: str):
-        time.sleep(self._SLEEP_SEC)
-        return {
-            "creation_datetime": _REMOTE_JOB_CREATION_TIMESTAMP,
-            "duration": _REMOTE_JOB_DURATION,
-            "failure_code": None,
-            "last_intermediate_results": None,
-            "msg": "ok",
-            "name": _REMOTE_JOB_NAME,
-            "progress": 1.0,
-            "progress_message": "Computing phases to apply (step 2)",
-            "start_time": _REMOTE_JOB_START_TIMESTAMP,
-            "status": "completed",
-            "status_message": None
-        }
-
-    def get_job_results(self, job_id: str):
-        time.sleep(self._SLEEP_SEC)
-        return {'results': json.dumps({
-            'results': serialize(_REMOTE_JOB_RESULTS),
-            'physical_perf': 1
-        })}
+from _mock_rpc_handler import MockRPCHandler, REMOTE_JOB_DURATION, REMOTE_JOB_RESULTS, REMOTE_JOB_CREATION_TIMESTAMP, \
```

Comment:
Wondering -
Should we split this file into 2 test files? `test_job.py` and `test_remote_job.py`? I kept trying to find import statements at the top hahaha :-D :-(